vendor/github.com/opencontainers/selinux/go-selinux/selinux.go

```diff
@@ -498,14 +580,14 @@ func DupSecOpt(src string) []string {
 		con["level"] == "" {
 		return nil
 	}
-	return []string{"label=user:" + con["user"],
-		"label=role:" + con["role"],
```

Comment:
label should not be in any SELinux specific section. this should be in the label section but not in the selinux specific section.